x/oracle/abci_test.go

```diff
@@ -22,8 +23,10 @@ import (
 )
 
 const (
-	displayDenom string = appparams.DisplayDenom
-	bondDenom    string = appparams.BondDenom
+	displayDenom     string = appparams.DisplayDenom
+	bondDenom        string = appparams.BondDenom
+	preVoteBlockDiff int64  = 2
+	voteBlockDiff    int64  = 3
```

Comment:
would these change if parameters change? 

if so, we need to derive them from the parameters

Reply:
done👍